WcaOnRails/lib/time_limit.rb

```diff
@@ -0,0 +1,36 @@
+# frozen_string_literal: true
+class TimeLimit
+  TEN_MINUTES_IN_CENTISECONDS = 10*60*100
+
+  attr_accessor :centiseconds, :cumulative
```

Comment:
Haha, there is no world that currently has `centiseconds` in it, `centiseconds` **is** the fresh new world. This is modelled closely after the structure and naming as defined in [the WCIF](https://docs.google.com/document/d/1hnzAZizTH0XyGkSYe-PxFL5xpKVWl_cvSdTzlT_kAs8/edit#heading=h.3z7evtp0blyd). I am up for changing this if you have a compelling alternative. Feel free to continue the discussion here, or over on the google doc.